.github/workflows/release.yml

```diff
@@ -219,45 +219,6 @@ jobs:
             exit 1
           fi
 
-      - name: Catalog - update veneer file
-        run: |
-          cd everest-catalog/tools
-
-          # configure userdata for commits
-          git config --global user.email "everest-ci@percona.com"
-          git config --global user.name "Everest RC CI triggered by ${{ github.actor }}"
-
```

Comment:
we should keep the step itself, the `cd` and the git config in order to have the `git tag $GH_TAG` which is still called

Reply:
we would need to rename the step though

Reply:
Regarding the `cd` I think we should be good because I merged this into the previous step and it already has the `cd everest-catalog` [here](https://github.com/percona/everest/blob/3faef882bba46f9a77f0710b35472d5af6c5ec96/.github/workflows/release.yml#L205).

Regarding the `git tag` requiring the `git config` I'm not 100% sure if it's required but it also doesn't hurt so I just added it back ine13dd1cad8d2c2b5f3e7a43cdfe8cdae30a6e0be